package.json

```
@@ -1275,7 +1275,7 @@
     "watch": "tsc --watch",
     "lint": "eslint ./ --ext ts && tsc --noEmit",
     "format": "prettier --check *.json src test",
-    "pretest": "npm run compile && del-cli ./assets/test/**/.build && tsc -p ./",
+    "pretest": "npm run compile && npm run esbuild && del-cli ./assets/test/**/.build",
```

```suggestion
    "pretest": "npm run compile-tests",
```
or just get rid of the "compile-tests" script (make sure Extension Tests debug config is updated)